lib/course_planner_web/controllers/course_controller.ex

```diff
@@ -30,17 +30,6 @@ defmodule CoursePlannerWeb.CourseController do
     end
   end
 
-  def show(conn, %{"id" => id}) do
-    case Repo.get(Course, id) do
-      nil ->
-        conn
-        |> put_status(404)
-        |> render(CoursePlannerWeb.ErrorView, "404.html")
-      course ->
-        render(conn, "show.html", course: course)
-    end
-  end
-
```

Comment:
Why is the `show` action being removed?

Reply:
Because it has only 2 fields that is already shown on the index list